ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/CommandTopicBackupImpl.java

```diff
@@ -46,18 +46,13 @@
  * complete backup of the command_topic is created.
  */
 public class CommandTopicBackupImpl implements CommandTopicBackup {
+
   private static final Logger LOG = LoggerFactory.getLogger(CommandTopicBackupImpl.class);
-  private static final Ticker CURRENT_MILLIS_TICKER = new Ticker() {
-    @Override
-    public long read() {
-      return System.currentTimeMillis();
-    }
-  };
```

Comment:
No need to use `Ticker` here. 

a) `Ticker` is marked as unstable
b) `Ticker.read()`'s java docs say it returns nanoseconds, not milliseconds as this impl does
c) A simple `Supplier<Long>` does what is needed.